### ISSUE TITLE
Memory / Parsing issue ?

### ISSUE THREAD

**Issue by PonteIneptique** [Capitains/jQuery.typeahead#3]
@balmas It seems that on http://www.perseids.org/apps/treebank you do not have access to the whole repository (type Martial as an example. Or Homer). I think we might want to investigate this one.

**Comment by balmas** [comment]
I'm pretty sure the problem is that some of the works are missing the `@xml:lang`  in the cts metadata (an oversight in Hook test?) They're all there in the inventory as retrieved but this line in the jquery.cts.typeahead causes them to be skipped:

`if((_this.settings.languages.length === 0) || (_this.settings.languages.length > 0 && $.inArray(inventory[textgroup][work][type][text].lang, _this.settings.languages) !== -1`
https://github.com/Capitains/jQuery.typeahead/blob/master/src/jquery.cts.typeahead.js#L332

**Comment by balmas** [comment]
hooktest bug entered as https://github.com/Capitains/HookTest/issues/32

**Comment by balmas** [comment]
yep, fixed Martial and it's there in the typeahead now. @PonteIneptique shall we close this and perhaps also enter bugs against the texts which are missing this in their metadata ?
